Java/Primitive Data Type/Literals/Literals_Video.md

```diff
@@ -0,0 +1,2 @@
+## Video Literals
+[Java_Literals_Video](https://drive.google.com/drive/folders/1egbVi5lELMBZ-B4WU-5mtkbq6kyE_GxU?usp=sharing)
```

Comment:
When I am clicking on the link, it is redirecting to a folder, with the file inside it.

Rather than this, can you change the link directly to the file (Inside the folder, right-click on the file, and select `get link` option, copy the link and change that with this)

Reply:
Yes...changing it

Reply:
I have done the required changes . Please check